### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ihorandrianov**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -3,16 +3,27 @@
...
+    .split(';')
+    .map(a => a.split(':').map(b => b.trim()));
+
+  const styles = {};
+
+  for (const style of stylesRaw) {
+    if (style.length === 2) {
+      styles[style[0]] = style[1];
+    }
+  }
```
Comment (Dikey945) [inline src/convertToObject.js:24]: try to use iteration methods, you can use reduce method in this case

Comment (Dikey945) [reply src/convertToObject.js:24]: > _ No description provided.
Good Job! 🔥

Comment (OleksiiBlezniuk) [inline src/convertToObject.js:24]: Let's use array methods here, reduce is best option here 🙂

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]

File: src/convertToObject.js [hunk 2]
```diff
@@ -3,16 +3,28 @@
...
+    .map(a => a.split(':').map(b => b.trim()));
+
+  const styles = stylesRaw.reduce((acc, [style, value]) => (style !== ''
+    ? {
+      ...acc,
+      [style]: value,
+    }
+    : {
+      ...acc,
+    }), {});
```
Comment (OleksiiBlezniuk) [inline src/convertToObject.js:25]: It's strange, better to use if in this case

**Review by OleksiiBlezniuk — APPROVED** [review]

**Pull request closed without merging**